test/data/repositories/rhel-88.json

```diff
@@ -2,17 +2,17 @@
   "x86_64": [
     {
       "name": "baseos",
-      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.8-20221025/",
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.8-20230301/",
```

Comment:
Please update all snapshots in this repo file if you are updating x86 repos. Otherwise we may run into broken dependencies and also would end up testing different things on different architectures.